application/forms/BpConfigForm.php

```diff
@@ -25,9 +25,15 @@ public function setup()
                 ),
                 array(
                     'validator' => 'Regex',
-                    'options' => array(
-                        'pattern' => '/^[a-zA-Z0-9](?:[a-zA-Z0-9 ._-]*)?[a-zA-Z0-9_]$/'
-                    )
+                    'options' => [
+                        'pattern' => '/^[a-zA-Z0-9](?:[\w\h.-]*)?[a-zA-Z0-9]$/',
```

Comment:
The trailing underscore was previously allowed. Was this a mistake here or is there actually a problem with a trailing underscore?

Reply:
I have added the missing validators without underscore. The old config will still work with a trailing underscore, but the user should not be able to create a new one. See below:

https://github.com/Icinga/icingaweb2-module-businessprocess/blob/6f1c95f32e397def69f19a564a46fc0064a46093/application/forms/AddNodeForm.php#L94